docs/content/docs/2.components/form-field.md

```diff
@@ -143,6 +143,31 @@ slots:
 This sets the `color` to `error` on the form control. You can change it globally in your `app.config.ts`.
 ::
 
+### Error Pattern
```

Comment:
The example should be on the `Form` component since it's not functional without a Form.

Reply:
@romhml @benjamincanac - do you think the entire thing should be moved to the `Form` page? or that there should be an example within the `form-field.md` page that shows it being used within a Form?

https://github.com/nuxt/ui/pull/5284#discussion_r2452307403

Reply:
alright @romhml  I readjusted so that the main example is on the Form component page, and the FormField just links to that